src/azul/terra.py

```diff
@@ -91,129 +93,119 @@
 log = logging.getLogger(__name__)
 
 
+class SourceType(enum.Enum):
+    bigquery = 'bigquery'
+    parquet = 'parquet'
+
+
+class SourceDomain(enum.Enum):
```

Comment:
```suggestion
class TDRSourceDomain(enum.Enum):
```

or nest in TDRSourceSpec